proposals/0221-multiple-modules.md

```diff
@@ -495,25 +502,24 @@ A simple Module Id based permission control is listed here.
 
-- `GetInteriorVehicleDataConsent` needs be extended to mobile api, so that an app can send a request to control/reserve a RC module after getting the SystemCapabilites and user's intent. 
-
-- Once the SDL receives a `GetInteriorVehicleDataConsent`, `SetInteriorVehicleData` or `ButtonPress`, if the requested RC module is exclusive to a user (`share=false`), without asking the driver, SDL shall grant the access only if the user's grid equals to or is within module service area. (The reason behind it is that a user shall be able to control his own seat without asking driver's permission, it does not make sense if a pop up is shown on driver's screen). Otherwise, SDL rejects the request.
+- `GetInteriorVehicleDataConsent` needs be extended to mobile API, so that an app can send a request to control/reserve a RC module after getting the SystemCapabilites and user's intent. 
 
-- If the requested RC module is shared among multiple users (`share=true`), all users located within the module's service area can potentially control the module without asking driver's permission.
-  - If the RC setting is auto deny, then only the first user can control the module. Whoever requests a free shared module can lock the module and use it until the user frees the module. 
-  - If the RC setting is auto allow, then every user can control the module. 
-  - If the RC setting is ask driver, SDL shall grant or deny the request following the current resource allocation rules: if module is free, allow access; if module is used by another user or app, and current app is in HMI FULL, SDL shall trigger or forward `GetInteriorVehicleDataConsent` to HMI. HMI shows a popup so that the driver can give permission to the app for accessing the requested module. 
-  - In order to reduce driver distractions, the default setting shall be auto allow.
+- Once the SDL receives a `GetInteriorVehicleDataConsent`, `SetInteriorVehicleData` or `ButtonPress`, if the requested RC module is exclusive to a user (`allowMultipleAccess=false`), without asking the driver, SDL shall grant the access only if the user's grid equals to or is within module service area. (The reason behind it is that a user shall be able to control his own seat without asking driver's permission, it does not make sense if a pop up is shown on driver's screen). Otherwise, SDL rejects the request.
```

Comment:
```suggestion
- Once SDL receives a `GetInteriorVehicleDataConsent`, `SetInteriorVehicleData` or `ButtonPress`, if the requested RC module is exclusive to a user (`allowMultipleAccess=false`), without asking the driver, SDL shall grant the access only if the user's grid equals to or is within module service area. (The reason behind this is that a user shall be able to control his own seat without asking driver's permission; it does not make sense if a pop up is shown on driver's screen). Otherwise, SDL rejects the request.
```

Reply:
removed "the"